support/scripts/configupdate

```diff
@@ -25,18 +25,12 @@ NL=$'\n'
 # Change of architecture?
 _comparesym "CONFIG_ARCH_" "${UK_CONFIG}" "${UK_CONFIG_OLD}" || UPDATES+="*** - CPU architecture changed${NL}"
 
-# Change of CPU options?
```

Comment:
Even after transiting, we should be able to properly handle re-compilation on updated compile command line. This means that CPU options and optimization should be covered. Actually, also the architecture should be covered (since we call a different compiler). I thought of even removing the architecture line but I don't know if we should keep it due external libraries. I am currently imagining a library that downloads different sources depending on the architecture. 🤔